### ISSUE TITLE
[BUG] Dark Mode Toggle appearing twice in 'Dashboard' Page

### ISSUE THREAD

**Issue by yashmandi** [subhadeeproy3902/BloxAI#70]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

The Dark Mode toggle button appears twice on the 'Dashboard', once in the Navbar and second, right next to search bar. It should be only once, in the Navbar. 
Would request you to assign me to fix this bug under GSSoC '24 :)
![Screenshot (205)](https://github.com/subhadeeproy3902/BloxAI/assets/132436131/49c50c65-51c1-4a54-8d0c-a50c3e8a9abc)

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

**Comment by subhadeeproy3902** [comment]
It was a minor fix @yashmandi . I have already done it !
